README.md

```diff
@@ -1,16 +1,20 @@
-FirebaseUI: Simplyify Mobile Development with Firebase
+FirebaseUI: Simplifying Mobile Development with Firebase
 ======================================================
 
-FirebaseUI is a library that allows developers to quickly build native mobile applications with Firebase by binding common UI views to Firebase references or queries. Currently FirebaseUI supports `ListViews`, `GridViews`, and `RecyclerViews` on Android through custom Firebase Adapters and `UITableViews` and `UICollectionViews` on iOS through custom Data Sources.
+FirebaseUI is a library that provides common UI elements when building apps. This includes a 
```

Comment:
Delete redundant `a` at the end of this line.

>[...] This includes ~~a~~

After this change, feel free to go ahead and merge. The rest looks good.